docs/wiki/00 - SERVER INSTALL AND CONFIGURATION/01 - LORIS Install/Macintosh/README.md

```diff
@@ -1 +1,290 @@
-Mac specific instructions
\ No newline at end of file
+### Mac is no longer supported as of 15.10.
+Nothing in this document is an official recommendation for LORIS but is only intended to be helpful!
+
+## Important Notes: 
+- We recommend installing LORIS on Ubuntu or CentOS as macOS is no longer supported for "production" as of v15.10
```

Comment:
```suggestion
- We recommend installing LORIS on Ubuntu or CentOS as macOS is no longer supported as of v15.10
```

I don't think it was ever really supported for production but we used to support developer installs